java/README.md

```diff
@@ -19,6 +19,7 @@ Please refer to the [Dataproc Templates (Java - Spark) README](java/README.md)
 * [GCSToJDBC](/java/src/main/java/com/google/cloud/dataproc/templates/gcs/README.md) (blogpost [link](https://medium.com/google-cloud/importing-data-from-gcs-to-databases-via-jdbc-using-dataproc-serverless-7ed75eab93ba))
 * [SnowflakeToGCS](/java/src/main/java/com/google/cloud/dataproc/templates/snowflake/README.md)
 * [KafkaToBQ](/java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md) (blogpost [link](https://medium.com/google-cloud/export-data-from-apache-kafka-to-bigquery-using-dataproc-serverless-4a666535117c))
+* [KafkaToGCS](/java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md)
```

Comment:
Same needs to be done in /README.md as well

Reply:
Changes done. Will push and resolve conversation.